documentation/web-components/integrating-a-web-component.asciidoc

```diff
@@ -16,27 +16,27 @@ To be able to use a Web Component in Vaadin you need:
 The Web Component's client-side files (typically JavaScript module files) are available using https://www.npmjs.com/[npm]. Vaadin 14 and above (Flow 2.0 +) supports npm and automatically installs and uses npm packages, and serves the static files to the browser.
 
 Since Vaadin 14.2 the _pnpm_ tool is used for package management instead of _npm_. This is done transparently. In case there is a reason to use npm instead of pnpm, you may disable pnpm
-(see <<../advanced/tutorial-all-vaadin-properties#,All Vaadin properties>>  or <<../production/tutorial-production-mode-advanced#,Advanced production mode topics>> sections).
-pnpm improves the package management in many ways, e.g. it allows to share the same packages between various
+(see <<../advanced/tutorial-all-vaadin-properties#,All Vaadin properties>> or <<../production/tutorial-production-mode-advanced#,Advanced production mode topics>>).
+pnpm improves the package management in many ways; for example, it allows to share the same packages between various
 projects and avoid downloading them every time for every project which also reduces the number of IO operations.
 
 == Integrating a JS Module into Vaadin
 
-While you can start from scratch and do it all manually, the easiest way is to use the https://vaadin.com/start/lts/component[Vaadin Add-on Starter]. This gives you:
+While you can start from scratch and do it all manually, the easiest way is to use the https://vaadin.com/start/latest/component[Vaadin Add-on Starter]. This gives you:
 
 * A project with all the necessary dependencies. 
 * An npm import for the selected component. 
 * A stub component Java class for your Web Component integration. 
 * A Maven profile that handles everything necessary to deploy the component to Vaadin Directory.
 
-*Example*: Annotations in the Java component class for a starter project for the https://github.com/PolymerElements/paper-slider[paper-slider] Web Component.
+*Example*: Annotations in the Java component class for a starter project for the https://github.com/material-components/material-components-web-components/tree/master/packages/slider[mwc-slider] Web Component.
 
 [source, java]
 ----
-@Tag("paper-slider")
-@NpmPackage(value = "@polymer/paper-slider",
-            version = "3.0.1")
-@JsModule("@polymer/paper-slider/paper-slider.js")
+@Tag("mwc-slider")
+@NpmPackage(value = "@material/mwc-slider",
+            version = "0.18.0")
```

Comment:
Non-blocker: this gives me the creeps slightly when something has not been graduated to `1.0` maturity ...